src/auth/strategies/jwt.strategy.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import { PassportStrategy } from '@nestjs/passport'
+import { ExtractJwt, Strategy } from 'passport-jwt'
+
+import { throwMissingEnvVar } from '../../common/utils/env.utils'
+
+@Injectable()
+export class JwtStrategy extends PassportStrategy(Strategy) {
+  constructor(configService: ConfigService) {
+    super({
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+      ignoreExpiration: false,
+      secretOrKey:
```

Comment:
Opinionated comment, but in case of a line break, I prefer to move the operators to the next line.
Typically, a developer reads the beginning of lines when getting familiar with the code, and the operator at the beginning of the line clearly signals that the was a line break thus it is a not an independent expression.
```suggestion
      secretOrKey: configService.get<string>('JWT_SECRET')
        || throwMissingEnvVar('JWT_SECRET'),
```

Reply:
Understanded, I will take into account that this is also possible